geom.js

```diff
@@ -3,6 +3,22 @@ class Rectangle {
     this.length = length;
     this.width = width;
   }
+
+  isSquare (length, width) {
+    if (this.length===this.width) {
+      return `True`;
```

Comment:
Returning the boolean `true` instead of a string will end up being more helpful here, so that you could do something like this in the future...

```js
if(squareInstance.isSquare){
  // run code that should happen if it is in fact a square
}
```